spec/components/search_spec.rb

```diff
@@ -1892,9 +1892,11 @@ def indexed_post(*args)
 
     it 'allows to define custom order' do
       expect(Search.new("advanced").execute.posts).to eq([post1, post0])
+
       Search.advanced_order(:chars) do |posts|
-        posts.reorder("(SELECT LENGTH(raw) FROM posts WHERE posts.topic_id = subquery.topic_id) DESC")
```

Comment:
@lis2 Which plugin was using this advanced order? I need to fix that plugin as the query will break.

Reply:
@tgxworld `discourse-voting` https://github.com/discourse/discourse-voting/blob/dd21880cc4bfbc0cf8496fcda851ac851e3431cf/plugin.rb#L113